R/model_LaTeX.R

```diff
@@ -26,25 +22,43 @@ SIR_LaTeX <- function(mu) {
 ## and only if, the function needs the value of the mu parameter should you give
 ## the function the formal parameter; if needed, then make the function call in
 ## server.R pass input$muValue also.
-SIRS_LaTeX <- function() {}
+SIRS_LaTeX <- function(mu) {
+  switch(mu + 1, 
+         helpText(
+           r"(Susceptible $$\frac{dS}{dt} = - \beta \frac{ S I}{N} + \xi{R}$$)",
+           r"(Infectious $$\frac{dI}{dt} = \frac{\beta S I}{N} - \gamma {I}$$)",
+           r"(Recovered $$\frac{dR}{dt} = \gamma {I} - \xi{R} $$)",
+           r"(Reproductive ratio $$R_0 =  \frac{\beta}{\gamma} S(0)^q$$)", #need to double check 
+           "q-Value $$\\begin{equation} x = \\begin{cases} 1 & \\text{, } frequency − dependent \\\\ 0 & \\text{, } density − dependent \\end{cases}\\end{equation} $$"
+           ),
+         helpText(
+           # want to make sure it is \nu{S}
+           r"(Susceptible $$\frac{dS}{dt} = \mu{N}- \beta \frac{ S I}{N} + \xi{R} - \nu{S}$$)", 
+           r"(Infectious $$\frac{dI}{dt} = \frac{\beta S I}{N} - \gamma {I} - \nu{I}$$)",
+           r"(Recovered $$\frac{dR}{dt} = \gamma {I} - \xi{R} - \nu{R}$$)",
+           r"(Reproductive ratio $$R_0 =  \frac{\beta}{\gamma} S(0)^q$$)", #need to double check 
+           "q-Value $$\\begin{equation} x = \\begin{cases} 1 & \\text{, } frequency − dependent \\\\ 0 & \\text{, } density − dependent \\end{cases}\\end{equation} $$"
+           )
+         )
+}
 
 SIRD_LaTeX <- function(mu) {
   switch(mu + 1,
     helpText(
       r"(Susceptible $$\frac{dS}{dt} = - \beta \frac{ S I}{N^q}$$)",
-      r"(Infectious $$\frac{dI}{dt} = \frac{\beta S I}{N^q} - \gamma I - \delta I $$)",
-      r"(Recovered $$\frac{dR}{dt} = \gamma I $$)",
-      r"(Dead $$\frac{dD}{dt} = \delta I $$)",
+      r"(Infectious $$\frac{dI}{dt} = \frac{\beta S I}{N^q} - \gamma {I} - \delta {I} $$)",
+      r"(Recovered $$\frac{dR}{dt} = \gamma {I} $$)",
+      r"(Dead $$\frac{dD}{dt} = \delta {I} $$)",
       r"(Reproductive ratio $$R_0 =  \frac{\beta}{\gamma + \delta} S(0)^q$$)",
-      "q-Value", br(), r"($$ 1, frequency-dependent $$)", br(), r"($$ 0, density-dependent $$)"
+      "q-Value $$\\begin{equation} x = \\begin{cases} 1 & \\text{, } frequency − dependent \\\\ 0 & \\text{, } density − dependent \\end{cases}\\end{equation} $$"
```

Comment:
Please use raw strings when writing $\LaTeX{}$. Inside the raw string you will only have to write $\LaTeX{}$, and won't need to escape backslash characters which are used to call macros and commands.